staging/com.scoreSmart/manifest.tt

```diff
@@ -0,0 +1,27 @@
+class @com.scoreSmart {
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+
+  monitorable list query standings(in req league_id: String #_[prompt="what league standings do you want out check out?"],
+                        out Team: String,
+                        out Rank: Number,
+                        out Form: String,
+                        out Points: Number)
+  #_[canonical="get league standings"]
+  #_[confirmation="league standings of $league_id"]
+  #_[formatted=[{type="text",text="${Rank}. Team: ${Team} | Pts:  ${Points} | Form: ${Form}"}]]
```

Comment:
In recent versions of Genie, the formatted annotation has been replaced with `#_[result]`. See the guide at https://wiki.almond.stanford.edu/en/thingpedia/guide/natural-language#result-phrases